scripts/Engineering/gui/engineering_diffraction/settings/settings_presenter.py

```diff
@@ -5,14 +5,16 @@
 #   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
 # SPDX - License - Identifier: GPL - 3.0 +
 # pylint: disable=invalid-name
-from os import path
+from os import path, getcwd
 from mantidqt.utils.observer_pattern import Observable
 
+CALIB_FOLDER = path.join(path.dirname(path.dirname(path.dirname(getcwd()))), "calib")
```

Comment:
Is there a better way than calling `dirname` three times (I don't know if there is!). I was wondnering if it would automatically be on the path (in the same way the Parameters.xml files are) and we can use the same filefinder stuff we use in algorithms?

Reply:
I'll have a look, might be possible to add the calib folder to the default data search directories

Reply:
did you find anything to help with this?